### PR TITLE
systemd packaging for kvmservice 

### PR DESCRIPTION
**Pull request by seswarrajan**

### CODE REVIEW

File: src/service/nfpm.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+priority: "extra"
+replaces:
+  - kvmservice
+provides:
+  - kvmservice
+maintainer: "Eswar Rajan Subramanian <eswar@accuknox.com>"
+description: |
+  KVM Service - Non-k8s control plane
+vendor: "kubearmor"
+homepage: "https://kubearmor.com"
```
Comment (nyrahul) [inline src/service/nfpm.yaml:15]:
```suggestion
homepage: "https://kubearmor.io"
```

**Review by nyrahul — CHANGES_REQUESTED** [review]

**Review by daemon1024 — COMMENTED** [review]
A few questions:
1. Does this expect kubearmor to be running already? or will it start kubearmor (in docker I presume) using genScript?
		- Will we need to add docker dependency as well?
2. CI for this will be handled in this PR or a seperate one?
3. Documentation to test out the changes?4.

**Comment by seswarrajan** [comment]
> A few questions:
> 
> 1. Does this expect kubearmor to be running already? or will it start kubearmor (in docker I presume) using genScript?
>    - Will we need to add docker dependency as well?
@daemon1024 kvmservice is on control-plane VM and does not require Kubearmor to be running already.  kvm-service is not dependent on kubearmor.

> 2. CI for this will be handled in this PR or a seperate one?
Planning to handle in separate PR. Please suggest @daemon1024 and @nyrahul  

> 3. Documentation to test out the changes?4.
Yet to be done.

**Comment by daemon1024** [comment]
> >      @daemon1024 kvmservice is on control-plane VM and does not require Kubearmor to be running already.  kvm-service is not dependent on kubearmor.

Makes sense. Reading the docs, only dependency is etcd. We should handle it's dependency and also start the etcd service as part of preinstall script if not already running. WDYT?

> > 2. CI for this will be handled in this PR or a seperate one?

It's fine either way imo.

**Comment by seswarrajan** [comment]
> > > ```
> > >  @daemon1024 kvmservice is on control-plane VM and does not require Kubearmor to be running already.  kvm-service is not dependent on kubearmor.
> > > ```
> 
> Makes sense. Reading the docs, only dependency is etcd. We should handle it's dependency and also start the etcd service as part of preinstall script if not already running. WDYT?
> 
> > > 2. CI for this will be handled in this PR or a seperate one?
> 
> It's fine either way imo.



> We should handle it's dependency and also start the etcd service as part of preinstall script if not already running. WDYT?
@daemon1024 I suggest to throw an error and notify user to install ETCD in their own method of installation. 
@nyrahul Please suggest a solution here,

**Review by nyrahul — APPROVED** [review]

**Pull request merged**